docs/README.md

```diff
@@ -6,35 +6,35 @@
 
 ## What is refgenie?
 
-Refgenie is full-service reference genome manager. It provides command-line and python interfaces to download pre-built genome assets like indexes. It can also build assets for custom genomes.
+Refgenie is full-service reference genome manager. It provides command-line and python interfaces to download pre-built reference genome "assets" like indexes used by different bioinformatics tools. It can also build assets for custom genome assemblies, and it facilitates systematic organization of, and access to, local genome "assets."
 
 ## What makes refgenie better?
 
 Refgenie provides programmatic access to a standard genome folder structure, so that software can easily swap from one genome to another. There are other similar projects, but Refgenie has a few advantages:
 
-1. **It provides a command-line interface to download individual resources**. Think of it as `GitHub` for reference genomes. You just type `refgenie pull -g hg38 -a bowtie2_index`.
+1. **It provides a command-line interface to download individual resources**. Think of it as `GitHub` for reference genomes. You just type `refgenie pull -g hg38 -a kallisto`.
 
-2. **It's scripted**. In case you need resources *not* on the server, such as for a custom genome, refgenie provides a `build` function to create your own: `refgenie build -i custom.fa.gz -a bowtie2_index`.
-
-3. **It includes a python API**. For tool developers, you use `cfg = refgenie.RefGenConf("genomes.yaml")` to get a python object with paths to any genome asset, *e.g.*, `cfg.hg38.bowtie2_index`.
+2. **It's scripted**. In case you need resources *not* on the server, such as for a custom genome, refgenie provides a `build` function to create your own: `refgenie build -i custom.fa.gz -a bowtie2`.
 
+3. **It includes a python API**. For tool developers, you use `cfg = refgenie.RefGenConf("genomes.yaml")` to get a python object with paths to any genome asset, *e.g.*, `cfg.hg38.kallisto`.
 
+4. When a new asset is downloaded, Refgenie can automatically update a local configuration file that acts as a sort of filesystem oracle for locally available genome assets. It's aware of the path to each resource that's been downloaded or otherwise declared.
 
 ## Quick example
 
 ### Downloading indexes and assets for a reference genome
 
 
 ```console
-refgenie pull --genome hg38 --asset bowtie2_index
+refgenie pull --genome hg38 --asset bowtie2
```

Comment:
it should be `bowtie2_index`, not `bowtie2`. The asset should be called `bowtie2_index`.

Reply:
So why doesn't this:
```
refgenie pull -c $REFGENIE -g hg38 -a bowtie2
```
fail?

Reply:
I've implemented the possibility of `path`  and asset name being two different strings:

https://github.com/databio/refgenconf/commit/f880e7f8004e4b9eda1c5535ca81746785630235

It suggested in the comments before.

Reply:
What I'm saying is only one of the following should be true--the docs can be left as-is, or the command I entered should fail.

Reply:
Ok, so @nsheff's comment actually will need to be addressed  in `refgenieserver`'s resources, right?

Since the name of the asset with the bowtie2 indexes is `bowtie2`, and the following fails and completes as expected:

```
[mstolarczyk@MichalsMBP refgenie](dev): refgenie pull -c ~/test/cfg.yaml -g rCRSd -a bowtie2_index
Starting pull for 'rCRSd/bowtie2_index'
Non-OK response (status 404) for URL: http://refgenomes.databio.org/asset/rCRSd/bowtie2_index
[mstolarczyk@MichalsMBP refgenie](dev): refgenie pull -c ~/test/cfg.yaml -g rCRSd -a bowtie2
Starting pull for 'rCRSd/bowtie2'
Replace existing (/Users/mstolarczyk/test/rCRSd/bowtie2.tar)? [y/N] N
```

Reply:
yes, the problem is the refgenie resources have named them as 'bowtie2' instead of as 'bowtie2_index".  I thought this was clear awhile ago, sorry for the confusion. we can leave them if you have strong feelings, I just don't understand why we would name the asset 'bowtie2'.

Reply:
I don't have a preference; I wanted to underscore the fact that both work, though, since it sounded like that may not have been the intent. If it is, though, that's fine.

Reply:
The `_index` way--at least for `hg38` and `bowtie2`--actually doesn't work at the moment for me, though. I'm behind the times, though--3.5.2 Python, 16.04 Ubuntu, local.
```
(py3.5) vr @ ~/code/refgenie
dev└─ $ refgenie pull -c $REFGENIE -g hg38 -a bowtie2_index
Determined genome config: /home/vr/refgenie.yaml
Starting pull for 'hg38/bowtie2_index'
Non-OK response (status 404) for URL: http://refgenomes.databio.org/asset/hg38/bowtie2_index
```

Reply:
Maybe this is subsumed under an issue with the server itself, though?

Reply:
the response is correct (see code example in: https://github.com/databio/refgenie/pull/28#discussion_r293025543), as there's no `bowtie2_index` asset for `hg38` genome (and it's tru for all genomes), try: http://refgenomes.databio.org/asset/hg38/bowtie2_index/archive

however now the error/message that we get is not precise, and I think it caused by the introduction of the [connection pre-check](https://github.com/databio/refgenconf/issues/32) that we do for `refgenie pull`